Makefile

```diff
@@ -0,0 +1,12 @@
+.PHONY: test
+test:
+	go test -v .
+
+.PHONY: lint
+lint:
+	golangci-lint run
+
```

Comment:
can we have a target to install golangci-lint and have it as a dependency for lint target ? 

see:
https://github.com/k8snetworkplumbingwg/sriov-cni/blob/868c852a4e83cf37c162f41c346a5a65ecd577fd/Makefile#L63